project_task_pull_request_state/models/project_task.py

```diff
@@ -0,0 +1,41 @@
+# Copyright Cetmix OU 2023
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0).
+from odoo import api, fields, models
+
+
+class ProjectTaskState(models.Model):
+    _inherit = "project.task"
+
+    pr_state = fields.Selection(
+        selection=lambda self: self._selection_pr_state(),
+        tracking=True,
+        copy=False,
+        string="PR State",
+        compute="_compute_pr_state",
+        precompute=True,
+        store=True,
+        readonly=False,
+    )
+
+    def _selection_pr_state(self):
```

Comment:
Do you anticipate that this could be plausibly extended in some cases? Please elaborate

Reply:
fixed